spec/services/csv/sales_log_csv_service_spec.rb

```diff
@@ -52,7 +52,7 @@
   end
 
   it "returns a csv with headers" do
-    expect(csv.first.first).to eq "id"
```

Comment:
Do we want an explicit test around differing headers for support / non-support? (It will happen I guess with the check that the csv matches the expected file)

Reply:
Hmm, yeah, I feel like it's implicitly tested, so it might not be the most useful thing to check (?)